decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Decidim
+  module Meetings
+    # The data store for a Meeting in the Decidim::Meetings component. It stores a
+    # title, description and any other useful information to render a custom meeting.
+    class Meeting < Meetings::ApplicationRecord
+      validates :title, presence: true
+      belongs_to :feature, foreign_key: "decidim_feature_id", class_name: Decidim::Feature
+      belongs_to :author, foreign_key: "decidim_author_id", class_name: Decidim::User
```

Comment:
I'd add a validation that the author & feature are from the same organization. (if we actually need the author)